<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yawpm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Shortcuts</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Create Application Launcher</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QListWidget" name="listWidget"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="prefixTab">
       <attribute name="title">
        <string>Prefixes</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QListWidget" name="prefixListWidget">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="addPrefixPushButton">
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="removePrefixPushButton">
                <property name="text">
                 <string>Remove</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_3">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Prefix Directory:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Architecture:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="prefixArchitectureLabel">
                 <property name="text">
                  <string>{ARCH}</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="prefixDirectoryLabel">
                 <property name="text">
                  <string>{DIRECTORY}</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTabWidget" name="tabWidget_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tabWidget_2Page1">
               <attribute name="title">
                <string>Configuration</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="2" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QToolButton" name="driveToolButton">
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset theme="system-file-manager">
                      <normaloff>.</normaloff>.</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>64</width>
                      <height>64</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="gameToolButton">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset theme="applications-games">
                      <normaloff>.</normaloff>.</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>64</width>
                      <height>64</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="installedToolButton">
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset theme="system-software-install">
                      <normaloff>.</normaloff>.</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>64</width>
                      <height>64</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="winetricksToolButton">
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset theme="system-software-update">
                      <normaloff>.</normaloff>.</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>64</width>
                      <height>64</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="cfgToolButton">
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset theme="preferences-system">
                      <normaloff>.</normaloff>.</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>64</width>
                      <height>64</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="0" column="0">
                 <widget class="QPushButton" name="runInPushButton">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Run program with this prefix</string>
                  </property>
                  <property name="icon">
                   <iconset theme="utilities-terminal">
                    <normaloff>.</normaloff>.</iconset>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QPushButton" name="pushButton">
                  <property name="text">
                   <string>Create Shortcut</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
